src/main/groovy/com/twcable/gradle/sling/SlingServerConfiguration.groovy

```diff
@@ -127,57 +122,65 @@ class SlingServerConfiguration {
         _active = active
     }
 
-    /**
-     * Returns the base URL to control a bundle.
-     */
-    @Nonnull
-    URI getBundleControlBaseUri() {
-        URI base = getBaseUri()
-        new URI(base.scheme, base.userInfo, base.host, base.port, '/system/console/bundles', null, null)
-    }
-
-    /**
-     * Returns the URL to use to do actions on a bundle.
-     */
-    @Nonnull
-    URI getBundleControlUriJson() {
-        URI base = getBundleControlBaseUri()
-        new URI(base.scheme, base.userInfo, base.host, base.port, "${base.path}.json", null, null)
-    }
-
-    /**
-     * Returns the URL to use to do actions on a CQ package.
-     */
-    @Nonnull
-    URI getPackageControlUri() {
-        URI base = getBaseUri()
-        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/service/.json', null, null)
-    }
+//    /**
+//     * Returns the URL to use to do actions on a CQ package.
+//     */
+//    @Nonnull
+//    URI getPackageControlUri() {
+//        URI base = getBaseUri()
+//        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/service/.json', null, null)
+//    }
+//
+//    /**
+//     * Returns the URL to list all the CQ packages.
+//     */
+//    @Nonnull
+//    URI getPackageListUri() {
+//        URI base = getBaseUri()
+//        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/list.jsp', null, null)
+//    }
+//
+//    /**
+//     * Returns the URL to download a CQ package.
+//     */
+//    @Nonnull
+//    URI getPackageDownloadUri() {
+//        URI base = getBaseUri()
+//        new URI(base.scheme, base.userInfo, base.host, base.port, '/crx/packmgr/download.jsp', null, null)
+//    }
+
+    /**
+     * Sets the property to the given value, doing type coercion if needed
+     */
+    void setTheProperty(final String propName, Object value) {
+        String realPropName = propName
+
+        // do any needed translations
+        switch (propName) {
+            case 'machinename': realPropName = 'machineName'; break
+            case 'retry.ms': realPropName = 'retryWaitMs'; break
+            case 'max.ms': realPropName = 'maxWaitMs'; break
```

Comment:
Above it's set to `propName`, so effectively there is.

Unfortunately, Groovy treats `switch` as a statement instead of an expression. If it were an expression (like Scala and other functional languages do) it would have looked like

```groovy
String realPropName = switch(propName) {
  case 'machinename': 'machineName'
  case 'retry.ms': 'retryWaitMs'
  case 'max.ms': 'maxWaitMs'
  default: propName
}
```

Much nicer, and beats what we often see with strung-together ternary operators...